### PR TITLE
changed area wording and added a button from list of pdfs

### PR DESCRIPTION
**Pull request by datanel**
area

pdf

### CODE REVIEW

File: Resources/translations/default.fr.yml [hunk 1]
```diff
@@ -128,8 +129,8 @@ area:
             label: Nom du secteur
     created: Le secteur a bien été ajouté
     deleted: Le secteur a bien été supprimé
-    pdf_list: Liste des pdf
-    pdf_modal_title: Pdf du secteur "%areaTitle%"
+    generate_pdf: Générer les pdf
+    pdf_modal_title: De quelle saison voulez-vous générer les PDF ?
```
Comment (dvdn) [inline Resources/translations/default.fr.yml]: homognéity ? upper case or lower case

File: Resources/translations/default.fr.yml [hunk 2]
```diff
@@ -83,6 +83,7 @@ season:
     start_date: Début
     end_date: Fin
     list: Liste des saisons
+    management: Gestion des saisons
     generate_pdf: Générer les pdfs
```
Comment (dvdn) [inline Resources/translations/default.fr.yml]: transduction doubloon ?

File: Resources/translations/default.fr.yml [hunk 3]
```diff
@@ -83,6 +83,7 @@ season:
...
     publication_and_generation: Souhaitez-vous également mettre à jour tous les pdfs de cette saison avant sa publication?
```
Comment (dvdn) [inline Resources/translations/default.fr.yml]: need a space after 'sa publication'

Comment (datanel) [inline Resources/translations/default.fr.yml]: this is not part of this PR but you edit it if you want

**Comment by dvdn** [comment]
Missing tests update
